src/erlfmt.erl

```diff
@@ -13,14 +13,18 @@
 %% limitations under the License.
 -module(erlfmt).
 
+-include_lib("stdlib/include/assert.hrl").
```

Comment:
You already mentioned this in the chat to me, but hopefully having a comment here makes it easy to tick off.

Reply:
Done!